astropy/vo/client/conesearch.py

```diff
@@ -472,7 +472,7 @@ def conesearch_timer(*args, **kwargs):
 
 
 def _local_conversion(func, x):
-    """Try ``func(x)`` and replace `~.exceptions.ValueError` with
+    """Try ``func(x)`` and replace `ValueError` with
```

Comment:
This is a hidden function, so it does not really matter. But :+1: for consistency.